### PR TITLE
feat(ruby): Document integrations - gems and patches

### LINKED ISSUE #11088: Document ruby patches
### Core or SDK?

Platform/SDK

### Which part? Which one?

ruby

### Description

* faraday
* graphql

default
* net/http
* redis
* puma

### PR DESCRIPTION
**Pull request by sl0thentr0py**
closes #11088

### CODE REVIEW

File: platform-includes/enriching-events/breadcrumbs/breadcrumbs-example/ruby.mdx [hunk 1]
```diff
@@ -6,23 +6,3 @@ crumb = Sentry::Breadcrumb.new(
 )
 Sentry.add_breadcrumb(crumb)
 ```
-
-Appropriate places to inject Breadcrumbs may be places like your HTTP library:
-
-```ruby
-# Instrumenting Faraday with a middleware:
-
-class SentryFaradayMiddleware
```
Comment (sl0thentr0py) [inline platform-includes/enriching-events/breadcrumbs/breadcrumbs-example/ruby.mdx:15]: removed this since it is now part of our faraday integration.

**Comment by sl0thentr0py** [comment]
cc @solnic

**Comment by vivianyentran** [comment]
Hi there, thanks for contributing to our docs! The docs team is focusing on other priorities at the moment, so it may take us another week or more to get to this PR. If you need us to review this sooner, please let us know so we can prioritize accordingly. Thanks for your patience!

**Comment by sl0thentr0py** [comment]
no rush :)

**Review by cleptric — APPROVED** [review]

**Pull request merged**